devops/resources/cloudflare-hit-rate.sh

```diff
@@ -2,6 +2,7 @@
 set -exuo pipefail
 
 cd $WORKSPACE/configuration
+pip install -r requirements-cloudflare.txt
```

Comment:
update based or requested change in configuration PR: Requirement file should be under `util/jenkins`